### PR TITLE
Certificate switch for futureofwork.gov.sg

### PR DESCRIPTION
**Pull request by shazlithebestie**

### CODE REVIEW

File: letsencrypt/futureofwork.gov.sg.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+server {
+    listen          443 ssl http2;
+    listen          [::]:443 ssl http2;
+    server_name     futureofwork.gov.sg;
+    ssl_certificate /etc/letsencrypt/live/futureofwork.gov.sg/fullchain.pem;
+    ssl_certificate_key     /etc/letsencrypt/live/futureofwork.gov.sg/privkey.pem;
+    return          301 https://futureofwork.gov.sg$request_uri;
+}
```
Comment (alexanderleegs) [inline letsencrypt/futureofwork.gov.sg.conf:8]: should this be
`return 301 https://www.futureofwork.gov.sg$request_uri;`?

**Review by kathleenkhy — APPROVED** [review]
lgtm

**Review by alexanderleegs — APPROVED** [review]
lgtm

File: letsencrypt/futureofwork.gov.sg.conf (hunk 1 above)
Comment (shazlithebestie) [reply letsencrypt/futureofwork.gov.sg.conf:8]: Edited!

**Pull request merged**